### DISCUSSION TITLE
ncnn-colorization

### DISCUSSION THREAD

**Discussion by magicse** [Tencent/ncnn#4186] [Ideas]
May be it will be interesting for someone. -> [ncnn-colorization](https://github.com/magicse/ncnn-colorization)

**Comment by magicse** [comment]
## Another one variant of colorization - [ncnn-colorization-siggraph17](https://github.com/magicse/ncnn-colorization-siggraph17)

![pict](https://user-images.githubusercontent.com/13585785/189329208-f081a6d6-d28c-41f0-ad59-320adf2f90ea.jpg)

**Comment by LRY89757** [comment]
Ａ wonderful work!⭐
